docker/docker-compose.prod.yml

```diff
@@ -0,0 +1,37 @@
+version: "3"
+services:
+  db:
+    image: postgres:13.9
+    container_name: phpreport-db
+    ports:
+      - "5432:5432"
```

Comment:
Does this port need to be exposed at all? Is it intended to be used by anything besides api and frontend?